modules/core/src/Controller/Exception.php

```diff
@@ -77,8 +77,8 @@ public function error(Request $request, string $code): Response
         $t->data['code'] = $code;
         $t->data['ts'] = $ts;
         $t->data['rp'] = $rp;
-        $t->data['tid'] => $tid;
-        $t->data['ctx'] => $ctx;
+        $t->data['tid'] = $tid;
+        $t->data['ctx'] = $ctx;
 
         $t->show();
```

Comment:
Shouldn't this be return $t ?

Reply:
Not sure what you mean.. This is adding variables to twig

Reply:
Ah, now I get it :)  You are absolutely right